examples/examples/untyped_publish_subscribe/publisher.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2024 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache Software License 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+// which is available at https://opensource.org/licenses/MIT.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use core::time::Duration;
+use iceoryx2::prelude::*;
+use std::alloc::Layout;
+
+const CYCLE_TIME: Duration = Duration::from_secs(1);
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let service_name = ServiceName::new("My/Funk/ServiceName")?;
+
+    let service = zero_copy::Service::new(&service_name)
+        .publish_subscribe()
+        .untyped::<&[u8]>()
```

Comment:
* Untyped API = sliced API with `[u8]`
    * `loan_uninit_with_layout`
* Sliced API in general 
    * `loan_uninit_slice`

```rust
    let service = zero_copy::Service::new(&service_name)
        .publish_subscribe()
        .type::<Whatever>()
        .open_or_create();

    let service = zero_copy::Service::new(&service_name)
        .publish_subscribe()
        .type::<[Whatever]>()
        .open_or_create(NUMBER_OF_ELEMENTS);

    let service = zero_copy::Service::new(&service_name)
        .publish_subscribe()
        .untyped()
        .open_or_create(Layout::from_size_align(128, 8));


```